src/main/java/org/jabref/model/entry/BibEntrySerializer.java

```diff
@@ -0,0 +1,42 @@
+package org.jabref.model.entry;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.InternalField;
+
+import java.lang.reflect.Type;
+import java.util.*;
+
+public class BibEntrySerializer implements JsonSerializer<BibEntry> {
+    @Override
+    public JsonElement serialize(BibEntry entry, Type typeOfSrc, JsonSerializationContext context) {
+        JsonObject obj = new JsonObject();
+
+        obj.addProperty("type", entry.getType().getName());
+        obj.addProperty("cite_key", entry.getCiteKeyOptional().orElse(""));
```

Comment:
Difficult decision here. Alternatives are `null` or to omit the field. It is OK for me to use the emptry string. Seems to be the JavaScript way.